lix/packages/sdk/src/discussion/start-discussion.ts

```diff
@@ -0,0 +1,68 @@
+import { Kysely } from "kysely";
+import type { LixDatabaseSchema } from "../database/schema.js";
+import { v4 } from "uuid";
+
+export async function startDiscussion(args: {
+	db: Kysely<LixDatabaseSchema>;
```

Comment:
If you pass lix as an argument, you can eliminate `currentAuthor` (and other state information that you might need in the future)

```ts
await createChangeDiscussion({
  changeIds,
  lix,
  body
})
```

```ts
export async function createChangeDiscussion(args: { lix, ... }){
  const currentAuthor = await lix.currentAuthor.get()
  // ...
}

```


---

Nice side-effect passing lix is what I sketched in https://github.com/opral/lix-sdk/issues/58#issuecomment-2342285024.

Reply:
EDIT: aha this function is used internally and then exposed via `lix.*` where you fill in the current author. If you agree on modelling lix API tree-shakable, it would prob be good to expose the two functions directly with a lix object. If not, just ship with `lix.*` API and we can revisit the tree-shakable discussion after the feature complete milestone.